src/help/faq/README.md

```diff
@@ -47,25 +49,49 @@ In the app, go to Settings > Content Settings.
 
 ---
 
-### I am on the public version of Paperback! When will other manga sources be added?
-Currently other manga sources are already available on the patreon version of the application. Sources on the public version of the app are expected to come most likely at the end of July.
+### How do I enable Hentai
+ * Sign into [MangaDex website](https://mangadex.org/) on a browser and go to your settings to set Hentai toggle to "**View All**".
```

Comment:
```suggestion
 * Sign into [MangaDex’s website](https://mangadex.org/) on a browse, go to your settings and set the Hentai toggle to "**View All**".
```

Reply:
What is a "browse" exactly. Could the `r` have unexpectedly disappear or was the modification intended?

Reply:
Oops, sorry about that! I’ve adjusted the suggestion to fit the proper grammar